app/models/course.rb

```diff
@@ -119,6 +119,7 @@ def new_draft_attributes
   validates :enrichments, presence: true, on: :publish
   validate :validate_enrichment_publishable, on: :publish
   validate :validate_enrichment
+  validate :validate_course_syncable, on: :sync
 
   after_validation :remove_unnecessary_enrichments_validation_message
```

Comment:
not sure if ths will throw a spanner in the works...

Reply:
Can you expand on this please?

Reply:
Discussed.